library/Kubernetes/Web/Conditions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/* Icinga for Kubernetes Web | (c) 2024 Icinga GmbH | AGPLv3 */
+
+namespace Icinga\Module\Kubernetes\Web;
+
+use ipl\Html\Attributes;
+use ipl\Html\HtmlDocument;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\I18n\Translation;
+use ipl\Web\Widget\Icon;
+use ipl\Web\Widget\TimeAgo;
+
+abstract class Conditions extends HtmlDocument
+{
+    use Translation;
+
+    abstract protected function getConditions(): iterable;
+
+    abstract protected function getVisual(string $status, string $type): array;
+
+    public function assemble()
```

Comment:
This method should `protected` as in the base class and the return type should be added.